src/ComboboxInput/__stories__/ComboboxInput.stories.js

```diff
@@ -3,41 +3,43 @@ import Menu from '../../Menu/Menu';
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import {
+    boolean,
+    select,
     withKnobs
 } from '@storybook/addon-knobs';
 
+const menu = () => (<Menu>
```

Comment:
This story doesn't work if is menu is a function, looks like you have to call it below `menu()` or just define it as `const menu = (<Menu>...`

Reply:
oops! Good catch!